### PR TITLE
global: refactoring

### PR DESCRIPTION
**Pull request by lnielsen**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,77 +1,62 @@
...
 script:
-  - "check-manifest --ignore .travis-\\*-requirements.txt"
-  - "sphinx-build -qnN docs docs/_build/html"
-  - "python setup.py test"
+  - "./run-tests.sh"
 
 after_success:
   - coveralls
-
-deploy:
```
Comment (jirikuncar) [inline .travis.yml]: can you keep it?

File: CHANGES.rst [hunk 2]
```diff
@@ -21,45 +21,10 @@
     waive the privileges and immunities granted to it by virtue of its status
     as an Intergovernmental Organization or submit itself to any jurisdiction.
 
+
 Changes
 =======
 
-Version 0.2.2 (released 2015-10-02)
```
Comment (jirikuncar) [inline CHANGES.rst]: please keep it :wink: thanks

File: docs/usage.rst [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    along with Invenio; if not, write to the
+    Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+    MA 02111-1307, USA.
+
+    In applying this license, CERN does not
+    waive the privileges and immunities granted to it by virtue of its status
+    as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+
+=======
```
Comment (jirikuncar) [inline docs/usage.rst]:
```
Usage
=====
```

**Comment by jirikuncar** [comment]
@lnielsen it looks good. After fixing tiny issues we can :shipit:

**Comment by lnielsen** [comment]
Ready :rocket:

**Pull request merged**